Containerfile.userinfo

```diff
@@ -0,0 +1,39 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.23.6-1747333074 as build
```

Comment:
Converter between AAP userinfo to the expected format (https://connect2id.com/products/server/docs/api/userinfo)